.github/workflows/lint-test.yaml

```diff
@@ -37,6 +37,62 @@ jobs:
       - run: cd ./ecosystem/indexer-server/typescript && yarn install
       - run: cd ./ecosystem/indexer-server/typescript && yarn lint
 
+  webplatform-lint:
+    runs-on: ubuntu-latest
+    env:
+      RAILS_ENV: test
+      NODE_ENV: test
+    defaults:
+      run:
+        working-directory: ecosystem/platform/server
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          working-directory: 'ecosystem/platform/server'
+          bundler-cache: true
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .node-version
+      - run: bundle exec bundler-audit --update
+      - run: bundle exec brakeman -q -w2
+      - run: bundle exec rubocop
+      - run: yarn install
+      - run: yarn run tsc
+
+  webplatform-test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11-alpine
+        ports:
+          - "5432:5432"
+        env:
+          POSTGRES_DB: web_platform_test
+          POSTGRES_USER: webplat
+          POSTGRES_PASSWORD: password
+    env:
+      RAILS_ENV: test
+      NODE_ENV: test
+      COMMUNITY_DB_USER: webplat
```

Comment:
instead of webplat, can we call it complat? Lets keep calling it community here for now

Reply:
Done.